anvio/genomedescriptions.py

```diff
@@ -1051,6 +1051,30 @@ class AggregateFunctions:
     including external genomes, internal genomes, and/or a genomes storage, and report
     a set of dictionaries that give access to the presence/absence and frequency
     of all functions annotated by a single source.
+
+    One fancy function in AggregateFunctions is `report_functions_per_group_stats`. For
+    instance, one could initiate the class in the following way to get a functions per
+    group stats output file for functioanl enrichment analysis:
+
+        >>> import argparse
+        >>> import anvio.genomedescriptions as g
+        >>> args = argparse.Namespace(external_genomes=external_genmes_path, annotation_source='KOfam')
+        >>> groups = {'adoles': ['B_adolescentis', 'B_adolescentis_1_11', 'B_adolescentis_22L', 'B_adolescentis_6', 'B_adolescentis_ATCC_15703'],
+                      'lactis': ['B_animalis', 'B_lactis_AD011', 'B_lactis_ATCC_27673', 'B_lactis_B420', 'B_lactis_BB_12', 'B_lactis_BF052'],
+                      'longum': ['B_longum', 'B_longum_AH1206', 'B_longum_BBMN68', 'B_longum_BORI', 'B_longum_CCUG30698', 'B_longum_GT15']}
+        >>> facc = g.AggregateFunctions(args, layer_groups=groups)
+        >>> facc.report_functions_per_group_stats(output_file_path)
```

Comment:
Hey @ivagljiva :) There is now a function that reports a file that is somewhat similar to the file format that goes to the enrichment analysis as input, but there are missing columns and I believe the column names will need to be updated.

If you send me an example file and I can fix the function to match to that.

Reply:
I am on it :)